students/Hebdowska_Grazyna/lesson_01_basic/tensDigit.py

```diff
@@ -0,0 +1,3 @@
+number = str(input())
+n = number[-2]
```

Comment:
There is no requirement that number must be >10 so it's possible to get out of index exception for number[-2].
Also snakify test doesn't pass with this slution

Example solution
# Read an integer:
number = int(input("provide number: "))
tens_digit = (number // 10) % 10
# Print a value:
print(tens_digit)